packages/cli/src/commands/tv.ts

```diff
@@ -12,7 +12,17 @@ export default (client: Client): Record<string, RTVCommand> => ({
     action: ({ fullscan }) => client.tv.list({ fullscan }),
     options: [['-s, --fullscan', 'scan server network (for Tizen only)', false]],
   },
-
+  
+  known: {
+    command: 'known',
+    description: 'List all known TVs',
+    action: ({ all, extra }) => client.tv.list({ showInvisible: all, additionalInfo: extra }),
```

Comment:
Did you mean `client.tv.getKnownTvs` here?

Reply:
fixed